### DISCUSSION TITLE
Greenfield API: exchange rates endpoint

### DISCUSSION THREAD

**Discussion by ndeet** [btcpayserver/btcpayserver#3837] [Technical Support] [answered]
We have a somehow [undocumented exchange endpoint](https://github.com/btcpayserver/btcpayserver-doc/issues/1124) that was inspired by BitPay but needs to be store specific /api/rates/?storeId=XYZ, e.g. https://testnet.demo.btcpay.tech/api/rates?storeId=46y7JuD7dUHEN1BxeAmtReA5zjUzSho7HVXVteQr1XUW

The discussion here is about if we want that endpoint also to be available in Greenfield API. I think it would make sense but eager to hear what you think.

**Comment by Kukks** [comment, marked as answer]
I agree, I implemented this way back when I wanted to make btcpay fully
compatible with the bitpay api. We should definitely make a variant of this
for Greenfield!
